repo-labels.json

```
@@ -0,0 +1,87 @@
+[
+  {
+    "color": "#eb6420",
+    "description": "This issue or pull request is blocked",
+    "name": "blocked"
+  },
+  {
+    "color": "#000000",
+    "description": "This issue or pull request will cause existing functionality to change",
+    "name": "breaking change"
+  },
+  {
+    "color": "#d73a4a",
+    "description": "This issue or pull request addresses broken functionality",
+    "name": "bug"
+  },
+  {
+    "color": "#07648d",
+    "description": "This issue will be advertised on code.gov's \"Open Tasks\" page (https://code.gov/open-tasks)",
+    "name": "code.gov"
+  },
+  {
+    "color": "#5319e7",
+    "description": "This issue or pull request involves improvements or additions to documentation",
+    "name": "documentation"
+  },
+  {
+    "color": "#cfd3d7",
+    "description": "This issue or pull request already exists",
+    "name": "duplicate"
+  },
+  {
+    "color": "#b005bc",
+    "description": "This issue is an epic",
+    "name": "epic"
+  },
+  {
+    "color": "#0e8a16",
+    "description": "Good for newcomers",
```

Update wording to align more closely with other descriptions:
```suggestion
    "description": "This issue is a good candidate for newcomers to work on",
```